docs/source/intro.rst

```diff
@@ -1,17 +1,18 @@
 Introduction
 ------------
 
-Goal of the tool
-^^^^^^^^^^^^^^^^
 
-Allow DataLad to work with the Open Science Framework (OSF). This allows for direct access to the OSF via a `git-annex <https://git-annex.branchable.com/>`_ `special remote <https://git-annex.branchable.com/special_remotes/>`_.
+Goal of the extension
+^^^^^^^^^^^^^^^^^^^^^
 
-What can I use this tool for
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This extension aims to allow DataLad to work with the Open Science Framework (OSF). This is done by creating a `git-annex <https://git-annex.branchable.com/>`_ `special remote <https://git-annex.branchable.com/special_remotes/>`_ that can transform datasets on the OSF into git-annex repositories and vice versa. Files in OSF storage could thus be consumed or exported fast and easily via git-annex or DataLad, and published to repository-hosting services (GitHub, GitLab, Bitbucket, ...) as lightweight repositories that constitute an alternative access to the data stored on the OSF - that is: you can git clone a repository from for example GitHub and get the data from the OSF from the command line or in your scripts.
```

Comment:
The same here:
While this was the project description as we pitched it, this is not what the tool now does. Instead of transforming OSF storage into git-annex repository, you can use this extension to use the OSF as a special remote to store data in the annex of a dataset. With this, you can datalad publish a dataset to GitHub or similar services and the data -- via a publication dependency -- to the OSF.